web_remember_tree_column_width/readme/CREDITS.md

```diff
@@ -0,0 +1,6 @@
+- Vauxoo
```

Comment:
This file can be removed in this version.

Reply:
Updated, Thanks